parsers/openapi.c

```diff
@@ -0,0 +1,361 @@
+/*
+*
+*   Copyright (c) 2016, Masatake YAMATO
+*   Copyright (c) 2016, Red Hat, K.K.
+*   Copyright (c) 2022, Vasily Kulikov
+*
+*   This source code is released for free distribution under the terms of the
+*   GNU General Public License version 2 or (at your option) any later version.
+*
```

Comment:
Could you add references for the target language or input like:
https://github.com/universal-ctags/ctags/blob/82154927e9a31a088f2fcc9b00e22d279d362b72/parsers/gdscript.c#L12

`git push --force` is welcome.